scan-command/src/test/java/io/ballerina/scan/utils/ScanUtilsTest.java

```diff
@@ -126,4 +120,174 @@ void testGenerateScanReportToProvidedDirectory() throws IOException {
                 .replace(WINDOWS_LINE_SEPARATOR, LINUX_LINE_SEPARATOR);
         Assert.assertEquals(result, expected);
     }
+
+    @Test(description =
+            "test method for loading configurations from a Scan.toml file in a Ballerina single file project")
+    void testloadScanTomlConfigurationsForSingleFileProject() {
+        Path ballerinaProject = testResources.resolve("test-resources")
+                .resolve("single-file-project-with-config-file").resolve("main.bal");
+        Project project = SingleFileProject.load(ballerinaProject);
+        String userDir = System.getProperty("user.dir");
+        System.setProperty("user.dir", ballerinaProject.toString());
+        ScanTomlFile scanTomlFile = ScanUtils.loadScanTomlConfigurations(project, printStream).orElse(null);
+        System.setProperty("user.dir", userDir);
+        Assert.assertNotNull(scanTomlFile);
+    }
+
+    @Test(description = "test method for loading configurations from a Scan.toml file")
+    void testloadScanTomlConfigurations() {
+        Path ballerinaProject = testResources.resolve("test-resources")
+                .resolve("bal-project-with-config-file");
+        Project project = BuildProject.load(ballerinaProject);
+        String userDir = System.getProperty("user.dir");
+        System.setProperty("user.dir", ballerinaProject.toString());
+        ScanTomlFile scanTomlFile = ScanUtils.loadScanTomlConfigurations(project, printStream).orElse(null);
+        System.setProperty("user.dir", userDir);
+        Assert.assertNotNull(scanTomlFile);
+        Set<ScanTomlFile.Analyzer> analyzers = scanTomlFile.getAnalyzers();
+        List<ScanTomlFile.Analyzer> analyzerList = new ArrayList<>(analyzers);
+        Assert.assertEquals(analyzerList.size(), 4);
+        ScanTomlFile.Analyzer analyzer = analyzerList.get(0);
+        Assert.assertEquals(analyzer.org(), "exampleOrg");
+        Assert.assertEquals(analyzer.name(), "exampleName");
+        analyzer = analyzerList.get(1);
+        Assert.assertEquals(analyzer.org(), "ballerina");
+        Assert.assertEquals(analyzer.name(), "example_module_static_code_analyzer");
+        Assert.assertEquals(analyzer.version(), "0.1.0");
```

Comment:
If we are asserting analyzers at multiple places would it make sense to extract these assertions to a function